### PR TITLE
[DEV-716]: Fix API SEND

### PR DESCRIPTION
**Pull request by jessc93**
#### List of Changes

Fix Api slection style

#### Motivation and Context

Fix Api slection style for SEND

#### How Has This Been Tested?

User can see updated api selection component in:  http://localhost:3000/send/api

#### Screenshots (if appropriate):
screenshot-localhost_3000-2023 07 25-12_19_11

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/_contents/send/api.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { sendApiPath } from '@/_contents/send/apiPath';
 export const sendApi: ApiData = {
   ...sendApiPath,
   product: send,
+  specURLsName: 'Documentazione API di Send',
```
Comment (kin0992) [inline apps/nextjs-website/src/_contents/send/api.ts:8]:
```suggestion
  specURLsName: 'Documentazione API di SEND',
```
SEND should be uppercase

File: apps/nextjs-website/src/components/molecules/ApiSection/ApiSection.styles.ts [hunk 2]
```diff
@@ -1,20 +1,27 @@
 export const styles = {
+  formControl: {
+    alignSelf: 'center',
+    '& .MuiFormLabel-root': {
+      color: 'white !important', // TODO: find a way to override this
```
[the remark this answers is not in the document]
Comment (MarcoPonchia) [reply apps/nextjs-website/src/components/molecules/ApiSection/ApiSection.styles.ts:5]: I tried but unfortunately MUI add additional class via javascript and without `!important` in style changes are overwritten.

File: apps/nextjs-website/src/components/molecules/ApiSection/ApiSection.styles.ts [hunk 3]
```diff
@@ -1,20 +1,27 @@
...
+    },
+  },
   select: {
     color: 'white',
-    border: '1px solid white',
     margin: 1,
-    '&:before': {
-      borderColor: 'white',
+    '& .MuiOutlinedInput-notchedOutline': {
+      borderColor: 'white !important', // TODO: find a way to override this
```
[the remark this answers is not in the document]
Comment (MarcoPonchia) [reply apps/nextjs-website/src/components/molecules/ApiSection/ApiSection.styles.ts:14]: Same here. MUI add additional class via javascript and without `!important` in style changes are overwritten.

File: apps/nextjs-website/src/components/molecules/ApiSection/ApiSection.styles.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (datalek) [reply apps/nextjs-website/src/components/molecules/ApiSection/ApiSection.styles.ts]: 👍 What do you think to add a comment about why the `!important` is required?

**Review by datalek — APPROVED** [review]

**Pull request merged**